packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -41,8 +42,7 @@ const bottomMenuItems: SidebarMenuItem[] = [
   {
     icon: <ListIcon Icon={TerminalIcon} />,
     title: 'Changelog',
-    path: 'https://changelog.daily.dev/',
-    target: '_blank',
+    path: '/sources/daily_updates',
```

Comment:
A friendly reminder that this could be in the extension and not just the webapp.
Make sure to support this case. We have some samples throughout the codebase